cdc/kv/shared_stream.go

```diff
@@ -196,13 +197,32 @@ func (s *requestedStream) run(ctx context.Context, c *SharedClient, rs *requeste
 
 		s.tableExclusives = make(chan tableExclusive, 8)
 		g.Go(func() error {
+			connAndClientsCache := make(map[SubscriptionID]*sharedconn.ConnAndClient)
```

Comment:
Please add a comment to indicate this cached is only use to release conn when this goroutine exits.